packages/loader/container-loader/package.json

```diff
@@ -105,5 +106,13 @@
     "sinon": "^7.4.2",
     "typescript": "~4.1.3",
     "typescript-formatter": "7.1.0"
+  },
+  "typeValidation": {
+    "version": "0.58.3000",
+    "broken": {
+      "0.58.2000":{
+        "ClassDeclaration_Container": {"backCompat": false}
```

Comment:
fyi @sdeshpande3 . suppression for #9654.  i will revert if we revert that change

Reply:
PR to revert `raiseContainerWarning` in main: https://github.com/microsoft/FluidFramework/pull/9675#issue-1186829775